staging/src/k8s.io/legacy-cloud-providers/aws/volumes.go

```diff
@@ -104,7 +104,7 @@ func (c *Cloud) checkIfAttachedToNode(diskName KubernetesVolumeID, nodeName type
 		// has been deleted
 		if err != nil {
 			fetchErr := fmt.Errorf("error fetching instance %s for volume %s", instanceID, diskName)
-			klog.Warning(fetchErr)
+			klog.Warning("fetching instance for volume error", "instanceID",instanceID, "volume", diskName)
```

Comment:
klog.InfoS("Fetching instance for volume error", "instanceID", instanceID, "volume", diskName)